apps/src/levelbuilder/skills/AccuracyDetails.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {useState} from 'react';
+import './skills.css';
+
+// Displays accuracy percent
+const AccuracyPercent: React.FC<{percentAccurate: number}> = ({
+  percentAccurate,
+}) => (
+  <div style={{marginTop: '16px', fontWeight: 'bold'}}>
+    Your accuracy is at {percentAccurate}%
```

Comment:
nit picky: I wonder if this is more like "AI evaluations were {percentAccurate}% accurate"